### PR TITLE
Enable --cache-to and --cache-from in devcontainer up

### PR DESCRIPTION
**Pull request by aledbf**
Enabling the "--cache-to" option when using devcontainer up avoids the need for separate build and up commands, saving around 10 seconds of execution time (introduced by the interaction with buildkit twice)

### CODE REVIEW

File: src/spec-node/containerFeatures.ts [hunk 1]
```diff
@@ -89,6 +89,10 @@ export async function extendImage(params: DockerResolverParameters, config: Subs
 			params.additionalCacheFroms.forEach(cacheFrom => args.push('--cache-from', cacheFrom));
 		}
 
+		if (params.buildxCacheFrom) {
```
Comment (samruddhikhandale) [inline src/spec-node/containerFeatures.ts:92]: Question: How is `buildxCacheFrom` different from the existing `additionalCacheFroms` property?

Comment (aledbf) [reply src/spec-node/containerFeatures.ts]: Is not. Fixed.

**Review by chrmarti — APPROVED** [review]
LGTM, thanks!

@samruddhikhandale Could you take another look?

**Review by samruddhikhandale — APPROVED** [review]
Looks great, thanks!

**Pull request merged**